### PR TITLE
Rigged boxing gloves

### PR DESCRIPTION
**Pull request by GoodWheatley**
## About the PR

This PR adds a new Passenger & Boxer-exclusive item, Rigged Boxing Gloves. These gloves have covertly installed weights inside them, causing them to do significant Blunt and Stamina damage per punch. I believe this is a better way to implement a disguised stun than the Electro Gloves PR because a) it takes 4 hits to stun someone instead of two, b) the boxing gloves sprite is much more visible than yellow gloves, and c) only one job on the station can reliably have this, unless somebody got lucky with a surplus.

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Rigged Boxing Gloves are now available in the uplink for 5 TC, if you're a Passenger or Boxer.

### CODE REVIEW

**Comment by EmoGarbage404** [comment]
kinda just tangentially related, but boxer is currently unused as Lighthouse was the last map to implement it and no one has added it to their map since (as far as I know).

**Comment by metalgearsloth** [comment]
If this is unused there's no point merging it

**Comment by EmoGarbage404** [comment]
> If this is unused there's no point merging it

it's available for passengers, which is kinda fine I guess.

**Comment by metalgearsloth** [comment]
Are boxing gloves mapped on every station? Seeing someone walking around with em would be highly suss if they were syndie items.

**Comment by EmoGarbage404** [comment]
boxing gloves are pretty common, yeah.

**Comment by GoodWheatley** [comment]
> kinda just tangentially related, but boxer is currently unused as Lighthouse was the last map to implement it and no one has added it to their map since (as far as I know).

future proofing in case a new map adds it back

**Comment by deltanedas** [comment]
reasons for tiders to wear boxing gloves
- easily accessible way of hiding fingerprints, if det actually does anything
- can stun with punches very good against flukies

File: Resources/Prototypes/Entities/Clothing/Hands/gloves.yml [hunk 1]
```diff
@@ -74,6 +74,29 @@
...
+  components:
+    - type: Sprite
+      sprite: Clothing/Hands/Gloves/Boxing/boxingblue.rsi
+      state: icon
+    - type: Clothing
+      sprite: Clothing/Hands/Gloves/Boxing/boxingblue.rsi
+    - type: Fiber
+      fiberMaterial: fibers-leather
+      fiberColor: fibers-blue
+    - type: FingerprintMask
```
Comment (AJCM-git) [inline Resources/Prototypes/Entities/Clothing/Hands/gloves.yml:91]: Just inherit ClothingHandsGlovesBoxingBlue and get rid of this

**Review by AJCM-git — CHANGES_REQUESTED** [review]
this may conflict with #13935 design-wise

**Comment by GoodWheatley** [comment]
> this may conflict with #13935 design-wise

The title for that is incorrect, the author changed the PR to just remove the combat bonus outright.

File: Resources/Prototypes/Entities/Clothing/Hands/gloves.yml [hunk 2]
```diff
@@ -74,6 +74,20 @@
     fiberColor: fibers-yellow
   - type: FingerprintMask
 
+- type: entity
+  parent: ClothingHandsGlovesBoxingBlue
+  id: ClothingHandsGlovesBoxingRigged
+  name: blue boxing gloves
+  description: Blue gloves for competitive boxing. Feels like they're heavier than usual...
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Clothing/Hands/gloves.yml:81]: the only way to validcheck should be the boxer being suspiciously robust not walk up, strip, shift click gloves

File: Resources/Prototypes/Entities/Clothing/Hands/gloves.yml [hunk 3]
```diff
@@ -74,6 +74,20 @@
...
+  components:
+  - type: StaminaDamageOnHit
+    damage: 30
+  - type: MeleeWeapon
+    attackRate: 1.5
+    damage:
+      types:
+        Blunt: 8
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Clothing/Hands/gloves.yml:89]: damage too since its an extra click

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by AJCM-git** [comment]
@GoodWheatley you coming back to this?

**Comment by GoodWheatley** [comment]
> @GoodWheatley you coming back to this?

It’s ready to merge

**Comment by AJCM-git** [comment]
> > @GoodWheatley you coming back to this?
> 
> It’s ready to merge

Then merge master to see if tests passes and resolve delta's reviews

**Pull request merged**